checkov/sast/engines/prisma_engine.py

```diff
@@ -34,6 +34,8 @@
 logger = logging.getLogger(__name__)
 
 REPORT_PARSING_ERRORS = "report_parsing_errors"
+SAST_CORE_FILENAME_PATTERN = re.compile(r"(\d+_\d+_\d+)_library\.(so|dll|dylib)$")
+SAST_CORE_URL_PATTERN = re.compile(r".*/(?P<name>v?\d+_\d+_\d+_library\.(so|dll|dylib))\?.*")
```

Comment:
```suggestion
FILE_NAME_PATTERN = re.compile(r"(\d+_\d+_\d+)_library\.(so|dll|dylib)")
SAST_CORE_FILENAME_PATTERN = re.compile(rf"{FILE_NAME_PATTERN.pattern}$")
SAST_CORE_URL_PATTERN = re.compile(r".*/{FILE_NAME_PATTERN.pattern}\?.*")
```

just something to reduce this tiny code duplication, if you want